src/Products/PlonePAS/plugins/ufactory.py

```diff
@@ -19,7 +19,7 @@
 try:
     from collections import OrderedDict
 except ImportError:
-    from ordereddict import OrderedDict
+    from ordereddict import OrderedDict  # type: ignore
```

Comment:
We do not need this fallback at all, since we are Python 3.6+ only with current master.

Reply:
Ah, good catch.  I opted to just back out these 2 changes.  See [the commit message for why](https://github.com/plone/Products.PlonePAS/pull/66/commits/9810cdead931b46029be31fb9566ad9421627b87) if interested.